embeddings/transformation/flair/corpus_transformations.py

```diff
@@ -0,0 +1,94 @@
+import abc
+import pprint
+import tempfile
+from itertools import groupby
+from operator import itemgetter
+from pathlib import Path
+from typing import Dict
+
+import datasets
+from flair.data import Corpus, FlairDataset
+
+from embeddings.transformation.transformation import Transformation
+from experimental.defaults import DATASET_PATH
+
+
+class CorpusTransformation(Transformation[datasets.DatasetDict, Corpus]):
+    HUGGING_FACE_SUBSETS = ["train", "validation", "test"]
```

Comment:
it looks like general split naming convention, maybe we will move it to the whole project constants?

Reply:
Yes, but these names are specific to HuggingFace datasets. Some other Datasets may use different naming convention.